website/styles/globals.tsx

```diff
@@ -1,45 +1,49 @@
 import { css } from '@emotion/core';
+import CDN from 'utils/cdnURL';
```

Comment:
it's unusual to import the default with a name different from the name of the file